submitted_models/ctu_cras_norlab_spot_sensor_config_3/launch/spawner.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'common.rb'
+
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw, _additionalSpawnPlugins='')
```

Comment:
It seems we've already accepted #841 which contains the [same change](https://github.com/osrf/subt/blob/master/submitted_models/ctu_cras_norlab_spot_sensor_config_1/launch/spawner.rb).  If this is a problem, we can fix it here and then submit a PR for that case as well.

Reply:
I think that this is a mistake. You can't easily reuse `spawner` because Ruby replaces the old one with the new one as soon as you load another source file using `require`. That's why there are `spawner2`, `_spawner` and similar. I think removing the argument from here shouldn't be a problem. But rather test it.